server/controllers/stock/index.js

```diff
@@ -48,13 +52,13 @@ function createStock(req, res, next) {
 
   const document = {
     uuid : uuid.v4(),
-    date : moment(new Date(params.date)).format('YYYY-MM-DD').toString(),
+    date : moment(new Date(params.date)).format('YYYY-MM-DD HH:mm:ss').toString(),
     user : req.session.user.id,
   };
 
   params.lots.forEach((lot) => {
         // lot expiration date
-    date = moment(new Date(lot.expiration_date)).format('YYYY-MM-DD').toString();
+    date = moment(new Date(lot.expiration_date)).format('YYYY-MM-DD HH:mm:ss').toString();
```

Comment:
Why can't this code use MySQL's date functions?

On my machine:
```js
const mysql = require('mysql');
mysql.escape(new Date());
// --> '\'2017-08-11 08:17:57.930\''
```

In my mind, the code is a lot clearer if you write:
```js
date = new Date(lot.expiration_date);
```
then if you write:
```js
date = moment(new Date(lot.expiration_date)).format('YYYY-MM-DD HH:mm:ss').toString();
```
since they will almost map to the same thing.

For example, on my machines:
```js
const mysql = require('mysql');
const moment = require('moment');

// some random date - this would be lot.expiration_date in your case
const date = new Date();

// log the Moment version
console.log(moment(new Date(date)).format('YYYY-MM-DD HH:mm:ss').toString());
// -> 2017-08-11 08:22:38

console.log(mysql.escape(date));
// -> '2017-08-11 08:22:38.245'
```

From the above, there are only two differences:
 1. MySQL's function includes apostrophes (`'`) to escape the query
 2. MySQL includes µs and Moment only includes milliseconds.